linker/shared/src/main/scala/org/scalajs/linker/analyzer/Analysis.scala

```diff
@@ -75,8 +76,8 @@ object Analysis {
     def isDataAccessed: Boolean
     def linkedFrom: Seq[From]
     def instantiatedFrom: Seq[From]
-    def methodInfos: scala.collection.Map[String, MethodInfo]
-    def staticMethodInfos: scala.collection.Map[String, MethodInfo]
+    def methodInfos(
+        namespace: MemberNamespace): scala.collection.Map[String, MethodInfo]
```

Comment:
Is the choice of exposing this as a method (not a nested `Map`) deliberate? It feels a bit odd. (I completely see that it is easier to _implement_ it this way, but the mere interface feels odd :S). (this is a bump of an old comment).